src/commands/account/default.ts

```diff
@@ -32,98 +33,51 @@ export class DefaultAccount extends SwankyCommand<typeof DefaultAccount> {
-        if (!accountData) {
-          throw new ConfigError(`Provided account alias ${chalk.yellowBright(args.accountAlias)} not found in system config`);
-        }
-        this.swankyConfig.defaultAccount = accountData.alias;
-        await this.storeConfig(this.swankyConfig, "global");
-        console.log(chalk.greenBright(`Default account set to ${chalk.yellowBright(this.swankyConfig.defaultAccount)} in system config`));
-      }
-    } else {
-      let choices: {
-        name: string,
-        value: { alias: string, systemConfig: boolean }
-      }[] = [];
-      if(isLocalConfigCheck()) {
-        this.swankyConfig.accounts.forEach((account: AccountData) => {
-          choices.push({
-            name: `${account.alias} (${account.address})`,
-            value: { alias: account.alias, systemConfig: false },
-          });
-        });
-      }
-      if (!isEnvConfigCheck() || flags.global) {
-        systemConfig.accounts.forEach((account: AccountData) => {
-          if (!choices.find((choice: any) => choice.value.alias === account.alias)) {
-            choices.push({
-              name: `${account.alias} (${account.address}) [system config]`,
-              value: { alias: account.alias, systemConfig: true },
-            });
-          }
-        });
-      }
-      await inquirer.prompt([
-        {
-          type: "list",
-          name: "defaultAccount",
-          message: "Select default account",
-          choices: choices,
-        },
-      ]).then((answers) => {
-        if (answers.defaultAccount.systemConfig) {
-          systemConfig.defaultAccount = answers.defaultAccount.alias;
-          this.storeConfig(systemConfig, "global");
-          console.log(chalk.greenBright(`Default account set to ${chalk.yellowBright(systemConfig.defaultAccount)} in system config`));
-        } else {
-          if (flags.global) {
-            const systemAccountData = systemConfig.accounts.find(
-              (account: AccountData) => account.alias === answers.defaultAccount.alias,
-            );
-            if (!systemAccountData) {
-              throw new ConfigError(`Provided account alias ${chalk.yellowBright(answers.defaultAccount.alias)} not found in system config`);
-            }
-            systemConfig.defaultAccount = answers.defaultAccount.alias;
-            this.storeConfig(systemConfig, "global");
-          }
-          this.swankyConfig.defaultAccount = answers.defaultAccount.alias;
-          this.storeConfig(this.swankyConfig, "local");
-          console.log(chalk.greenBright(`Default account set to ${chalk.yellowBright(this.swankyConfig.defaultAccount)}`));
-        }
+    const accountAlias = args.accountAlias ?? (await this.promptForAccountAlias(config));
+    this.ensureAccountExists(config, accountAlias);
+
+    const configBuilder = new ConfigBuilder(config);
+    configBuilder.setDefaultAccount(accountAlias);
+
+    try {
+      await this.storeConfig(configBuilder.build(), configType);
```

Comment:
If configType is `local`, we will store in local config all accounts from both configs. But we shouldn't share system accounts with to project's config